### PR TITLE
1.23.1 Release Notes

### PR DESCRIPTION
**Pull request by thedebugger**
1.23.1 pre-release is in progress

### CODE REVIEW

File: content/en/news/releases/1.23.x/announcing-1.23.1/index.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: Announcing Istio 1.23.1
+linktitle: 1.23.1
+subtitle: Patch Release
+description: Istio 1.23.1 patch release.
+publishdate: 2024-09-09
+release: 1.23.1
+---
+
+This release note describes what is different between Istio 1.22.3 and 1.22.4.
```
Comment (dhawton) [inline content/en/news/releases/1.23.x/announcing-1.23.1/index.md:10]:
```suggestion
This release note describes what is different between Istio 1.23.0 and 1.23.1.
```

File: content/en/news/releases/1.23.x/announcing-1.23.1/index.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** an issue where controller-assigned IPs did not respect per-proxy DNS capture the same way that ephemeral auto-allocated IPs did.
+  ([Issue #52609](https://github.com/istio/istio/issues/52609))
+
+- **Fixed** an issue where Waypoints required DNSProxy to be enabled in order to consume auto-allocated IPs
```
Comment (dhawton) [inline content/en/news/releases/1.23.x/announcing-1.23.1/index.md:19]:
```suggestion
- **Fixed** an issue where waypoints required DNS proxy to be enabled in order to consume auto-allocated IPs.
```

File: content/en/news/releases/1.23.x/announcing-1.23.1/index.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  ([Issue #52746](https://github.com/istio/istio/issues/52746))
+
+- **Fixed** an issue where the `ISTIO_OUTPUT` `iptables` chain was not removed with `pilot-agent istio-clean-iptables` command.
+  ([Issue #52835](https://github.com/istio/istio/issues/52835))
+
+- **Fixed** an issue causing any `portLevelSettings` to be ignored in DestinationRules for waypoints.
```
Comment (dhawton) [inline content/en/news/releases/1.23.x/announcing-1.23.1/index.md:25]:
```suggestion
- **Fixed** an issue causing any `portLevelSettings` to be ignored in `DestinationRule`s for waypoints.
```

File: content/en/news/releases/1.23.x/announcing-1.23.1/index.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  ([Issue #52532](https://github.com/istio/istio/issues/52532))
+
+- **Removed** writing kubeconfig to CNI net directory.
```
Comment (dhawton) [inline content/en/news/releases/1.23.x/announcing-1.23.1/index.md:28]:
```suggestion
- **Removed** writing `kubeconfig` to CNI net directory.
```

File: content/en/news/releases/1.23.x/announcing-1.23.1/index.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  ([Issue #52315](https://github.com/istio/istio/issues/52315))
+
+- **Removed** `CNI_NET_DIR` from the `istio-cni` configmap, as it now does nothing.
```
Comment (dhawton) [inline content/en/news/releases/1.23.x/announcing-1.23.1/index.md:31]:
```suggestion
- **Removed** `CNI_NET_DIR` from the `istio-cni` `ConfigMap`, as it now does nothing.
```

File: content/en/news/releases/1.23.x/announcing-1.23.1/index.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+  ([Issue #52315](https://github.com/istio/istio/issues/52315))
+
+- **Removed** a change in Istio 1.23.0 causing regressions for ServiceEntries with multiple addresses defined.
+Note: the reverted change did fix an issue around missing addresses (#51747), but introduce a new set of issues.
+The original issue can be worked around by creating a Sidecar resource.
+  ([Issue #52944](https://github.com/istio/istio/issues/52944)),([Issue #52847](https://github.com/istio/istio/issues/52847))
```
Comment (dhawton) [inline content/en/news/releases/1.23.x/announcing-1.23.1/index.md:37]:
```suggestion
- **Removed** a change in Istio 1.23.0 causing regressions for `ServiceEntries` with multiple addresses defined.
  Note: the reverted change did fix an issue around missing addresses (#51747), but introduce a new set of issues.
  The original issue can be worked around by creating a sidecar resource.
  ([Issue #52944](https://github.com/istio/istio/issues/52944)),([Issue #52847](https://github.com/istio/istio/issues/52847))
```

**Review by dhawton — COMMENTED** [review]
Note: You'll need to open a manual cherrypick to release-1.23 and modify https://github.com/istio/istio.io/blob/release-1.23/data/args.yml#L5 with a make gen.

**Comment by wilsonwu** [comment]
Any process problem? This PR is not merged, but the cherry-pick done.

**Comment by dhawton** [comment]
It is blocked by [#155](https://github.com/istio/istio.io/pull/15595)

File: content/en/news/releases/1.23.x/announcing-1.23.1/index.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
+---
+title: Announcing Istio 1.23.1
+linktitle: 1.23.1
+subtitle: Patch Release
+description: Istio 1.23.1 patch release.
+publishdate: 2024-09-09
```
Comment (dhawton) [inline content/en/news/releases/1.23.x/announcing-1.23.1/index.md:6]:
```suggestion
publishdate: 2024-09-10
```

**Review by dhawton — APPROVED** [review]

**Pull request merged**